lib/reek/ast/sexp_extensions/send.rb

```diff
@@ -38,10 +37,6 @@ def object_creation_call?
           method_name == :new
         end
 
-        def visibility_modifier?
-          VISIBILITY_MODIFIERS.include?(method_name)
-        end
-
```

Comment:
To be more precise with: 

>> The more we put into contexts the better.

I believe our SexpExtensions should only **know** what you can see with the bare eye when looking at the corresponding Sexp. 
Everything that is slightly more complex should be put into `contexts` I believe.